charts/airbyte/templates/gcs-log-creds-secret.yaml

```diff
@@ -1,7 +1,7 @@
-# TODO: Make this more configurable or support an existing secret
 apiVersion: v1
 kind: Secret
 metadata:
-  name: gcs-log-creds
+  name: {{ include "common.names.fullname" . }}-gcs-log-creds
+type: Opaque
 data:
-  gcp.json: ""
+  gcp.json: "{{ .Values.logs.gcs.credentialsJson }}"
```

Comment:
This was also one that was there from before. I left it because it's mounting the contents as a volume rather than as env vars. I can merge it in too but as far as I remember it means that the file then can't be dynamically updated in the pod if the secret value updates, which in its current form it can.

Reply:
That makes sense, Let's keep it that way then!